src/egon/data/datasets/sanity_checks.py

```diff
@@ -769,8 +771,8 @@ def egon_power_plants_pv_roof_building():
             dataset = config.settings()["egon-data"]["--dataset-boundary"]
 
             if dataset == "Schleswig-Holstein":
-                # since the required data is missing for a SH run, it is implemented
-                # manually here
+                # since the required data is missing for a SH run, it is
+                # implemented manually here
                 total_2035 = 84070
                 sh_2035 = 2700
```

Comment:
Can't this value be obtained from scenario data? How fragile is this, e.g. does it still work for the new MaStR data?

Reply:
Done by accessing the nep scenario csv as the value for Germany is not represented within the db in a SH run